sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/FrameworkDetector.scala

```diff
@@ -77,4 +85,6 @@ object FrameworkDetector {
     val buf = mutable.Buffer.empty[String]
     def log(msg: Any): Unit = buf += msg.toString
   }
+
+  private val consoleFrameworkPrefix = "@framework:"
```

Comment:
Could you make this more verbose? Something like `@scalajs-test-framework-detector:`? This way someone that sees the string knows where it belongs to.